CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## WLED changelog
 
+#### Build 2411150
+-   WLED 0.15.0-rc1 release
```

Comment:
When making a release, update with all PRs and commits that happened after b7.

Reply:
Done